pkg/storage/export/virt-exportserver/exportserver_test.go

```diff
@@ -78,6 +79,14 @@ func newTestServer(token string) *exportServer {
 }
 
 var _ = Describe("exportserver", func() {
+	tmpDir := "/tmp"
+
+	BeforeEach(func() {
+		fs := afero.NewMemMapFs()
+		err := fs.Mkdir(tmpDir, 0755)
+		Expect(err).ToNot(HaveOccurred())
+	})
```

Comment:
Aren't you missing https://github.com/spf13/afero?tab=readme-ov-file#step-3-use-it-like-you-would-the-os-package?

Reply:
Aw you are right, so this will require a slightly more complex change on the server. Probably not worth using the package and just make a way to skip the readiness probe in unit tests. Thanks for the review!